.github/workflows/e2e.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         working-directory: e2e/python
         env:
           PYTHONPATH: ${{ github.workspace }}/e2e/python
+          FORCE_COLOR: 2
```

Comment:
saw in https://github.com/chalk/supports-color/issues/106

i think we could also handle ourselves via https://github.com/chalk/supports-color/issues/106#issuecomment-1970212713